icons/14x14_dollar_filled.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g opacity="0.84">
```

Comment:
This line should be removed because we're taking care of the opacity value on build time.